multipass/Linux.md

```diff
@@ -30,20 +31,21 @@ The primary instance is a Multipass virtual machine that is configured to be use
 
 The primary instance automatically mounts the $HOME directory (files in this directory are shared between the host and the instance), and it comes with Multipass's default specs: 1GB of RAM, 5GB of disk, and 1 CPU.
 -->
-
-From the application launcher, let's start Multipass. In Ubuntu, press the super key and type Multipass, or find Multipass in the Applications panel in the lower left of the desktop.
+Start Multipass from the application launcher. In Ubuntu, press the super key and type "Multipass", or find Multipass in the Applications panel on the lower left of the desktop.
 
 ![|800x450](https://assets.ubuntu.com/v1/949aa05e-mp-linux-1.png) 
 
-Once we've launched the application, we should see the Multipass tray icon in the upper right section of the screen:
+Once you've launched the application, you should see the Multipass tray icon on the upper right section of the screen.
 
 ![|688x52](https://assets.ubuntu.com/v1/5ec546da-mp-linux-2.png) 
 
-Let's click on the icon, then on "Open Shell". 
+Click on the **icon**, then select **Open Shell**. 
```

Comment:
I would not capitalise "icon" as it's not a UI label or button, just a reference to the Multipass icon. Also, since it's a very short sentence, I'd simplify it removing the comma:

```suggestion
Click on the Multipass icon and select **Open Shell**. 
```